practico_01/ejercicio-05.py

```diff
@@ -4,4 +4,7 @@
 
 # Resolver utilizando listas y el operador in.
 def es_vocal(letra):
-    pass
+    if letra == 'a' or letra == 'e' or letra == 'i' or letra == 'o' or letra == 'u':
+        return True
+    else:
+        return False
```

Comment:
@giovamarr hay que hacerlo utilizando listas y el operador in.
El else no es necesario.
Se podria hacer algo como 

```
vocales = ['a','e','i','o','u']
 return letra in vocales
```